search/includes/classes/class-queue.php

```diff
@@ -1080,18 +1080,25 @@ public function get_average_queue_wait_time() {
 
 		$table_name = $this->schema->get_table_name();
 
-		$average_wait_time = $wpdb->get_var(
+		$queue_stats = $wpdb->get_row(
+			// Cannot prepare table name. @codingStandardsIgnoreStart
 			$wpdb->prepare(
-				"SELECT FLOOR( AVG( TIMESTAMPDIFF( SECOND, queued_time, NOW() ) ) ) AS average_wait_time FROM $table_name WHERE 1" // Cannot prepare table name. @codingStandardsIgnoreLine
+				"SELECT
+					FLOOR( AVG( TIMESTAMPDIFF( SECOND, queued_time, NOW() ) ) ) AS average_wait_time,
+					FLOOR( MAX( TIMESTAMPDIFF( SECOND, queued_time, NOW() ) ) ) AS longest_wait_time,
+					COUNT( * ) AS queue_count
+				FROM $table_name
+				WHERE 1"
```

Comment:
Do we need `WHERE 1` here? AFAIK this would translate to always true.

Reply:
That's correct, but it's also a pattern that's used across WP, you can spot `WHERE 1` and `WHERE 1=1` in various parts of the core.